zhaquirks/sonoff/swv.py

```diff
@@ -56,5 +62,13 @@ class AttributeDefs(BaseAttributeDefs):
         translation_key="water_supply",
         fallback_name="Water supply",
     )
+    .switch(
+        CustomSonoffCluster.AttributeDefs.auto_close_when_water_shortage.name,
+        CustomSonoffCluster.cluster_id,
+        off_value=0,
+        on_value=30,
+        translation_key="auto_close_when_water_shortage",
+        fallback_name="Auto close when water shortage",
```

Comment:
"Water shortage auto-close" would be slightly less text (and get rid of the "when"):
```suggestion
        fallback_name="Water shortage auto-close",
```

Reply:
Thanks, I think it is a good idea to make the label shorter. I have checked, now it fits on the panel.